crates/sui-core/src/execution_driver.rs

```diff
@@ -72,46 +83,116 @@ pub async fn execution_process(
         };
         authority.metrics.execution_driver_dispatch_queue.dec();
```

Comment:
Is this still the right place to decrement `execution_driver_dispatch_queue`, or after acquiring permit?

Reply:
Yes, this is inside the dispatch task, e.g. when we did put task in the queue, and we need to decrement counter when we process the message